MobiusCore/Source/BackwardsCompatibility.swift

```diff
@@ -182,18 +182,6 @@ public extension Connectable {
     typealias OutputType = Output
 }
 
-public extension BrokenConnection {
```

Comment:
These are moved back into the main implementation, since all of `BrokenConnection` is deprecated now